to_production.txt

```diff
@@ -1,12 +1,13 @@
 # New
-
+ofl/notoserifhentaigana # https://github.com/google/fonts/pull/7327
```

Comment:
There was an issue with primary_script in sandbox.  I think that we might still need to solve this issue before the prod push?

@simoncozens to confirm that this is not meant as a fix for the issue in the production push here?

Reply:
cc @nathan-williams

Reply:
Let's remove the Noto Serif Hentatigana family from the to_prod list for now so that we can continue to move the other families.  Ty!